File: src/main/java/com/rs/java/game/area/multi/Islands.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class Islands extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Isles Multi";
	}

	public Islands() {
		super(9276, 9532, 8763, 8253, 8509, 8252, 8508);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/Islands.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.kotlin.game.world.area.Area

class Islands : Area(9276, 9532, 8763, 8253, 8509, 8252, 8508) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Isles Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
